src/nvim/testdir/test87.in

```diff
@@ -3,7 +3,7 @@ Tests for various python features.     vim: set ft=vim :
 STARTTEST
 :so small.vim
 :set noswapfile
-:if !has('python3') | e! test.ok | wq! test.out | endif
+:if !has('python3') || has('nvim') | e! test.ok | wq! test.out | endif
```

Comment:
test86 and test87 should just be deleted. The python client has its own test suite. cc @tarruda